### PR TITLE
chore: adapt node cmd to app/cosmos

### PR DESCRIPTION
**Pull request by Bidon15**
After upgrade of #736, we need to adapt our cmd to be compliant with what app/cosmos wants

This is especially needed for our keys cmds for the current state

A separate issue to refactor out this part is needed 
cc: @renaynay

### CODE REVIEW

File: cmd/celestia/main.go [hunk 1]
```diff
@@ -51,4 +68,19 @@ var rootCmd = &cobra.Command{
...
+		initClientCtx, err := client.ReadPersistentCommandFlags(initClientCtx, cmd.Flags())
+		if err != nil {
+			return err
+		}
+		initClientCtx, err = config.ReadFromClientConfig(initClientCtx)
+		if err != nil {
+			return err
+		}
+
+		if err := client.SetCmdClientContextHandler(initClientCtx, cmd); err != nil {
```
Comment (vgonkivs) [inline cmd/celestia/main.go:81]: `if err = ...`

Comment (Bidon15) [reply cmd/celestia/main.go]: Good Catch! Thanks you 🙏

File: cmd/celestia/main.go [hunk 2]
```diff
@@ -36,12 +47,18 @@ func main() {
 }
 
 func run() error {
-	return rootCmd.ExecuteContext(cmd.WithEnv(context.Background()))
+	cfg := sdk.GetConfig()
+	cfg.SetBech32PrefixForAccount(app.Bech32PrefixAccAddr, app.Bech32PrefixAccPub)
+	cfg.Seal()
```
Comment (liamsi) [inline cmd/celestia/main.go:52]: I still need to look into this.

**Review by liamsi — APPROVED** [review]

**Review by vgonkivs — APPROVED** [review]

**Comment by Wondertan** [comment]
Closing in favor of https://github.com/celestiaorg/celestia-node/pull/757 with agreement of @Bidon15. The https://github.com/celestiaorg/celestia-node/pull/757 has commit from this PR

**Pull request closed without merging**